relayer/relayer.go

```diff
@@ -28,8 +30,10 @@ import (
 )
 
 const (
-	maxSubscribeAttempts   = 10
-	maxResubscribeAttempts = 1000_000
+	maxSubscribeAttempts = 10
+	// TODO increase number of resubscribe attempts when we are able to process missed blocks and
+	// refresh the chain config on reconnect.
```

Comment:
```suggestion
	// TODO attempt to resubscribe in perpetuity once we are able to process missed blocks and
	// refresh the chain config on reconnect.
```

Suggest the following change to reflect the desired function. When we do add that, we should explicitly loop forever, rather than kicking off a long-running loop, even if they are in practice the same.